src/DoubleArraySender.cpp

```diff
@@ -0,0 +1,45 @@
+#include "networktables/NetworkTable.h"
+#include "networktables/NetworkTableInstance.h"
+#include "networktables/DoubleArrayTopic.h"
+
+#include <string>
+#include <iostream>
+#include <vector>
+
+/*
+Right now, the table is /SmartDashboard so we can visualize the values sent from the Orin.
+Later, this should be changed to a new table, such as /Orin or /Vision, to avoid populating the SmartDashboard with unnecessary values
+*/
+
+class DoubleArraySender{
+  private:
+    nt::NetworkTableInstance inst;
```

Comment:
please name private variables in a class with a trailing underscore, .e.g inst_ instead of inst, to follow the google style guide and make the code easier to read.  Reference: https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
fixed